src/OptimismGovernorV5.sol

```diff
@@ -294,16 +266,6 @@ contract OptimismGovernorV5 is
         _;
     }
 
-    function initialize(IVotesUpgradeable _votingToken, address _manager) public initializer {
-        __Governor_init("Optimism");
-        __GovernorCountingSimple_init();
-        __GovernorVotes_init(_votingToken);
-        __GovernorVotesQuorumFraction_init({quorumNumeratorValue: 30});
-        __GovernorSettings_init({initialVotingDelay: 6575, initialVotingPeriod: 46027, initialProposalThreshold: 0});
-
-        manager = _manager;
-    }
-
     function _execute(
         uint256, /* proposalId */
         address[] memory, /* targets */
```

Comment:
Why is execution skipped? Aren't there non module `execute()` calls that will rely on this?

I'm sure I'm missing something here but haven't trouble reminding myself why this works.

Reply:
OH was the idea that all execution is skipped and the Optimism team runs it manually?

Reply:
Precisely. They will eventually enable onchain execution, but for now everything is manual